README.md

```diff
@@ -67,6 +67,13 @@ mysql> grant all privileges on *.* to 'hyfuser'@'localhost';
 mysql> create database userdb;
 ```
 
+**Step 3: Installing MySQL drivers for node**
+
+We want to use MySQL with JavaScript and to this end, we use `mysqljs` driver.
+* Use `node install mysql` command to install them.
+* Run `node connection-test.js` from VScode (Windows/MAC) or the terminal (Linux/MAC).
```

Comment:
We can be clearer here, by mentioning that the `connection-test.js` file can be found within the directory. Students can navigate there using the CLI and then execute the command.

Reply:
Mentioned in the new commit